app/adapters/external/cache/adapter.py

```
@@ -131,10 +134,28 @@ async def ping(self) -> bool:
+        """패턴에 맞는 키 목록을 반환합니다."""
+        try:
+            client = self._get_client()
+            keys = client.keys(pattern)
+            # bytes를 문자열로 변환
+            if isinstance(keys, list):
+                return [
+                    key.decode("utf-8") if isinstance(key, bytes) else str(key)
+                    for key in keys
+                ]
+            else:
+                return []
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)